src/components/RecipeDescription/RecipeDescription.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { Children, Fragment } from 'react';
+import styles from './RecipeDescription.module.scss';
+import Text from 'components/Text';
+
+export type RecipeDescriptionProps = {
+    children: React.ReactNode;
+};
+
+const RecipeDescription: React.FC<RecipeDescriptionProps> = ({ children }) => {
+    return (
+        <Text view='p-16' className={styles.description__text}>
+            {Children.map(children, (child) => (
+                <Fragment>{child}</Fragment>
+            ))}
```

Comment:
хм, не видел такого подхода раньше... в [доке](https://react.dev/reference/react/Children) сказано, что это не особо надёжное решение
>Using Children is uncommon and can lead to fragile code.

почему бы не сделать по простому?
```suggestion
            {children}
```